rust-keybroker/keybroker-server/src/main.rs

```diff
@@ -198,9 +202,13 @@ struct Args {
     endpoint: Option<String>,
 
     /// The URL where the verifier can be reached
-    #[arg(long, default_value = "http://veraison.test.linaro.org:8080")]
+    #[arg(long, default_value = "https://veraison.test.linaro.org:8080")]
```

Comment:
It would be unconventional to do HTTPs on port 8080. I guess we need to know what the SSL deployment of the Linaro verifier looks like before we can be sure about this, but it seems more likely that it would be 443 or 8443 or something like that.

Reply:
Fixed in the next iteration of the patch !